command/daemon.go

```diff
@@ -476,7 +476,12 @@ func createValueStore(ctx context.Context, cfgIndexer config.Indexer) (indexer.I
 
 	switch cfgIndexer.ValueStoreType {
 	case vstoreStorethehash:
-		return storethehash.New(ctx, dir, storethehash.GCInterval(time.Duration(cfgIndexer.GCInterval)))
+		return storethehash.New(
+			ctx,
+			dir,
+			storethehash.GCInterval(time.Duration(cfgIndexer.GCInterval)),
```

Comment:
What would happen if bitesize is changed on existing data? 

We probably should error out, and maybe a TODO to do a migration ?

Reply:
https://github.com/filecoin-project/go-indexer-core/issues/62